### PR TITLE
Us03/doing exercises drag drop type UI

### PR DESCRIPTION
**Pull request by aduongdnai**
-  timer count down
- show how many exercises left, 
- edit logic for checking answer (right or not and calculate total score by currentRightAnswers variable)
- button go to next question.

### CODE REVIEW

File: Assets/Timer.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            DisplayTime(timeValue);
+        }
+    }
+    void DisplayTime(float timeToDisplay)
+    {
+        if (timeToDisplay < 0)
+        {
+            timeToDisplay = 0;
+        }
+        float minutes=Mathf.FloorToInt(timeToDisplay / 60);
```
Comment (QUanlE79) [inline Assets/Timer.cs:39]: should have spaces

File: Assets/Scripts/GlobalVariable.cs [hunk 2]
```diff
@@ -11,4 +11,6 @@ public class GlobalVariable
     public const string DragDropType = "DragDrop";
     public const string InputType = "Input";
     public const string MultipleChoiceType = "ABCD";
+    //PlayerPref variable string
+    public const string userID = "uid";
```
Comment (QUanlE79) [inline Assets/Scripts/GlobalVariable.cs:15]: Global constant variable should format uppercase and sparate by underscore, ex: `USER_ID = "uid"`

**Review by QUanlE79 — APPROVED** [review]
GTG

**Review by thaihiepnguyen — COMMENTED** [review]
I left comments.

Please help me check them.

Thank you.

**Pull request merged**